test/appframework_aws/c3/appframework_aws_test.go

```diff
@@ -911,7 +911,7 @@ var _ = Describe("c3appfw test", func() {
 			// Create App framework Spec
 			appSourceName := "appframework-shc-" + testenv.RandomDNSName(3)
 			appSourceVolumeNameShc := "appframework-test-volume-shc-" + testenv.RandomDNSName(3)
-			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopeClusterWithPreConfig, appSourceName, s3TestDirShc, 60)
+			appFrameworkSpecShc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameShc, enterpriseApi.ScopePremiumApps, appSourceName, s3TestDirShc, 60)
```

Comment:
The appframework spec for the ES app has other parameters such as `premiumAppsProps` as well as `esDefaults` etc.. Are we planning to add this later to test various values?

Reply:
+1

Reply:
Added to the spec later